src/sql/util.py

```diff
@@ -252,7 +252,7 @@ def support_only_sql_alchemy_connection(command):
 
 
 def fetch_sql_with_pagination(
-    table, offset, n_rows, sort_column=None, sort_order=None
+    table, offset, n_rows, sort_column=None, sort_order=None, with_clause=""
```

Comment:
it's considered best practice to set empty values as None, inside the function's  body you can assign another default value if you want:

```python
def something(arg=None):
    arg = arg or "" # if the value passed is None, False, or any other "empty" value, then arg will have the value of ""
```

Reply:
Made the change